<ui version="4.0" >
 <class>UpdateCenter</class>
 <widget class="QDialog" name="UpdateCenter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Update center</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/buttons/images/transmit_blue.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lblInfoList" >
     <property name="text" >
      <string>Available updates:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="lsvUpdates" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>243</green>
           <blue>197</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>243</green>
           <blue>197</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>243</green>
           <blue>197</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="whatsThis" >
      <string>List with all available updates</string>
     </property>
     <property name="accessibleName" >
      <string>Available updates list</string>
     </property>
     <property name="accessibleDescription" >
      <string>List with all available updates</string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="indentation" >
      <number>20</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblUpdateSate" >
       <property name="text" >
        <string>&lt;b>Update state:&lt;/b> Stopped</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>241</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblDownloadedSize" >
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text" >
        <string>0 bytes (0 bytes)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QProgressBar" name="pgbUpdate" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="Line" name="line" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>4</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>4</height>
      </size>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblxVSTVersion" >
       <property name="text" >
        <string>Current xVST version: &lt;b>%1&lt;/b></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>44</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="btnUpdate" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="whatsThis" >
          <string>Download and install the selected updates from the list</string>
         </property>
         <property name="accessibleName" >
          <string>Download and install button</string>
         </property>
         <property name="accessibleDescription" >
          <string>Download and install the selected updates from the list</string>
         </property>
         <property name="text" >
          <string>Install updates</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="whatsThis" >
          <string>Close without installing updates</string>
         </property>
         <property name="accessibleName" >
          <string>Cancel button</string>
         </property>
         <property name="accessibleDescription" >
          <string>Close without installing updates</string>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lsvUpdates</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
